tests/features/poetry_translations.feature

```diff
@@ -0,0 +1,62 @@
+@api @translation
+Feature: Poetry translations
+  In order to easily have multilingual content
+  As a translator
+  I need to be able to translate content using the Poetry service
+
+  @cleanup:tmgmt_job @cleanup:tmgmt_job_item @poetry
+  Scenario: Translate content.
+    Given oe_demo_translatable_page content:
+      | title    | field_oe_demo_translatable_body |
+      | My title | My body                         |
+    And I am logged in as a user with the "translator" role
+    When I visit "the content administration page"
+    And I click "My title"
+    Then I should see the link "Translate"
+
+    # Request the translation for two languages
+    When I click "Translate"
+    And I select the languages "Bulgarian, German" in the language list
+    And I press "Request DGT translation for the selected languages"
+    Then I should see "Send request to DG Translation for My title in Bulgarian, German"
+
+    When I fill in "Requested delivery date" with "05/04/2050"
+    And I fill in the the "first" "Author" field with "john"
```

Comment:
Not to be dealt with here, but I think the Poetry translator should require this fields, since DGT will require them from us as well.

Reply:
You mean the translator config entity? I tend to disagree a bit. Why should there be site defaults? The form for sending to Poetry has them required, sure, cause that is what is asked by Poetry. But if there are no site defaults, values must be provided at that moment.

Reply:
Fair enough